app/src/main/java/com/eveningoutpost/dexdrip/NFCReaderX.java

```diff
@@ -474,13 +480,27 @@ protected void onPostExecute(Tag tag) {
             }
         }
 
+        void LogLibre2StartStreaming(ENABLE_STREAMING success, String extraData) {
```

Comment:
Wouldn't a simple `boolean` value be sufficient here, would it? Or do you plan to extend the enum in the future to reflect fine granular streaming states?